src/test/java/com/redhat/red/build/finder/DistributionAnalyzerTest.java

```diff
@@ -77,6 +81,14 @@ public void verifyType() throws IOException {
 
     @Test
     public void verifyCacheClearance() throws IOException {
+        String osName = System.getProperty("os.name");
+
+        // XXX: Skip on Windows due to <https://issues.apache.org/jira/browse/VFS-634>
+        if (osName != null && osName.startsWith("Windows")) {
+            LOGGER.warn("Skipping test verifyCacheClearance() since ${os.name} is: {}", osName);
+            return;
```

Comment:
Wouldn't a Junit Assume guard be better here?

Reply:
Fixed in #71. Thanks.